### ISSUE TITLE
Introduce B2B, C2B and B2C payments. Introduction of such modes of payments will facilitate trade and e-commerce in general.

### ISSUE THREAD

**Issue by Timoh97** [twitter/the-algorithm#1401]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by GammaSoul** [comment]
Yes this!!! I'm tired of sending cash with PayPij... Half the time, people just shoot the pigeon out of the sky and take my money!!!!! I wish there were a multitude of easier and secure payment services online to choose from.....

**Comment by Sqaaakoi** [comment]
i would totally trust paying money through a service that actively supports people wanting to commit genocide

**Comment by farazkaleemmalik** [comment]
> i would totally trust paying money through a service that actively supports people wanting to commit genocide

this but unironically
